semantic-ast/semantic-ast.cabal

```diff
@@ -37,7 +37,10 @@ common haskell
 
 library
   import:              haskell
-  exposed-modules:     Marshal.JSON
+  exposed-modules:
+    AST.Element
+    AST.Traversable1
+    Marshal.JSON
```

Comment:
1. `AST.Element` was moved here from `semantic-tags`.
2. `AST.Traversable1` is a new module, but its contents were likewise moved from `semantic-tags`.
3. We should rename `Marshal.JSON` to `AST.Marshal.JSON` or something like that, but I haven’t done that in this PR. cc @aymannadeem

Reply:
added to my todos in #468